contracts/tests/HotfixClaimRevenue.t.sol

```diff
@@ -0,0 +1,334 @@
+
+        // 16_687_230: claimRevenue (called by thomas)
+        _rollAndWarpToBlock(16_687_230); 
+        vm.startPrank(THOMAS);
+        emit log_string("=> claimRevenue()");
+        vm.expectRevert(ISpigot.NoRevenue.selector);
+        claimed = spigot.claimRevenue(BORROWER_REVENUE_EOA, USDC, bytes(""));
+        vm.stopPrank();
+
+
+        // 16_687_712: claimAndRepay
+        _rollAndWarpToBlock(16_687_712); 
+        emit log_string("=> claimAndRepay()");
+        vm.expectRevert(ISpigot.ClaimFailed.selector);
+        line.claimAndRepay(USDC,hex"d9627aa400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000149970000000000000000000000000000000000000000000000000128e269a9e9abf4100000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000000000000000000000006b175474e89094c44da98b954eedeac495271d0f869584cd000000000000000000000000e9039a6968ed998139e023ed8d41c7fa77b7ff7a0000000000000000000000000000000000000000000000b9a844386663f6ab7a");
+
+        // 16_692_741: transfer 15 USDC to spigot
+        _rollAndWarpToBlock(16_692_741); 
+        vm.startPrank(BORROWER_REVENUE_EOA);
+        emit log_string("=> Transferring revenue to spigot");
+        IERC20(USDC).transfer(address(spigot), 15e6);
+        assertEq(IERC20(USDC).balanceOf(address(spigot)), spigotOperatorTokens + 15e6);
+        spigotOwnerTokens = spigot.getOwnerTokens(USDC);
+        spigotOperatorTokens = spigot.getOperatorTokens(USDC);
+        emit log_named_uint("owner tokens USDC after transfer", spigotOwnerTokens);
+        emit log_named_uint("operator tokens USDC after transfer", spigotOperatorTokens);
+        emit log_named_uint("line USDC balance", IERC20(USDC).balanceOf(lineAddress));
+        emit log_named_uint("spigot USDC balance", IERC20(USDC).balanceOf(address(spigot)));
+        vm.stopPrank();
+
+
+        // 16_693_128: claimRevenue
+        _rollAndWarpToBlock(16_693_128); 
+        vm.startPrank(borrower);
+        emit log_string("=> claimRevenue()");
+        uint256 previousSpigotOperatorTokens = spigot.getOperatorTokens(USDC);
+        claimed = spigot.claimRevenue(BORROWER_REVENUE_EOA, USDC, bytes(""));
+        emit log_named_uint("claimed", claimed);
+        spigotOwnerTokens = spigot.getOwnerTokens(USDC);
+        spigotOperatorTokens = spigot.getOperatorTokens(USDC);
+        expectedOwnerTokens = (claimed * revenueSplit) / 100;
+        emit log_named_uint("owner tokens USDC after claimRevenue", spigotOwnerTokens);
+        emit log_named_uint("operator tokens USDC after claimRevenue", spigotOperatorTokens);
+        emit log_named_uint("line USDC balance", IERC20(USDC).balanceOf(lineAddress));
+        emit log_named_uint("spigot USDC balance", IERC20(USDC).balanceOf(address(spigot)));
+        assertEq(spigotOwnerTokens, expectedOwnerTokens);
```

Comment:
shouldnt owner tokens be messed up too since the bug affects total revenue claimed, not just the operator tokens (but is caused by operator token amount)